### ISSUE TITLE
6.3.1 EE broken LDAP sync with license limit reached

### ISSUE THREAD

**Issue by shiryov** [RocketChat/Rocket.Chat#30301]
### Description:

When start LDAP background synchronization manually or by period, it does not run.

### Steps to reproduce:

1. Start background LDAP sync

### Expected behavior:

 LDAP background synchronization sync completed

### Actual behavior:

New users are not added, information about existing ones is not updated. Error in logs: error-license-user-limit-reached

### Server Setup Information:

- Version of Rocket.Chat Server: 6.3.1
- Operating System: Oracle Linux Server release 8.6
- Deployment Method: docker
- Number of Running Instances: 16
- DB Replicaset Oplog: yes
- NodeJS Version: v14.21.3
- MongoDB Version: 5.0.15 / wiredTiger (oplog Enabled)

### Client Setup Information

- Desktop App or Browser Version: Chrome 114
- Operating System: Windows, Mac, Linux
 
### Additional context
EE License active
![image](https://github.com/RocketChat/Rocket.Chat/assets/666827/ffc2c823-21e8-4f11-a81b-00198fcf065a)

![image](https://github.com/RocketChat/Rocket.Chat/assets/666827/2b8e8d7c-4c0d-41e0-b2b7-4008051b59f6)

The workaround is to launch the old instance 6.2.10 and start synchronization on it

### Relevant logs:

```
type=errorClass, message=The maximum number of users has been reached. [error-license-user-limit-reached], stack=Error: The maximum number of users has been reached. [error-license-user-limit-reached]
    at ee/server/startup/seatsCap.ts:37:10
    at /app/bundle/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:43:40
 => awaited here:
    at Function.Promise.await (/app/bundle/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/promise_server.js:56:12)
    at lib/callbacks.ts:318:12
```

**Comment by ankar84** [comment]
@sampaiodiego @pierre-lehnen-rc please check that issue

**Comment by pierre-lehnen-rc** [comment]
Already fixed in develop, we'll be patching it ASAP.

**Comment by shiryov** [comment]
Same on 6.3.3

**Comment by alarischu** [comment]
Bump. This is an important feature.

**Comment by alarischu** [comment]
> Already fixed in develop, we'll be patching it ASAP. You can also disable the setting to update existing users as a an alternative temporary fix.

Can you please refer to a commit in development that fixes this issue. I might need to patch it manually.

**Comment by pierre-lehnen-rc** [comment]
Fixed by #30232
